source/dnode/mnode/impl/src/mndMain.c

```diff
@@ -796,6 +797,12 @@ int32_t mndStart(SMnode *pMnode) {
       return -1;
     }
     mndSetRestored(pMnode, true);
+  } else {
+    if (sdbPrepare(pMnode->pSdb) != 0) {
```

Comment:
```suggestion
    if (sdbPrepare(pMnode->pSdb) != 0) {
```
这个函数里面有create和prepare transaction，会不会有问题？

Reply:
这个函数是在 open mnode 之后的 start mnode 调用，trans 模块已经初始化完成了，我觉得没问题。